code/game/gamemodes/clock_cult/clock_items/clockwork_slab.dm

```diff
@@ -410,7 +422,10 @@
 				data["tier_info"] = "<font color=#B18B25><i>Unlock these optional scriptures by converting another servant or if [DisplayPower(APPLICATION_UNLOCK_THRESHOLD)] of power is reached..</i></font>"
 
 	data["selected"] = selected_scripture
-
+	data["scripturecolors"] = "<font color=#DAAA18>Scriptures in <b>yellow</b> are related to construction and building.</font><br>\
```

Comment:
it's pretty obvious, even now, that scriptures that used ansibles are long-ranged and power-related.